definitions/procedures/packages/update_all_confirmation.rb

```diff
@@ -10,7 +10,10 @@ def run
       if @packages.nil? || @packages.empty?
         question = "\nWARNING: No specific packages to update were provided\n" \
           "so we are going to update all available packages.\n" \
-          "To Upgrade to next version use 'foreman-maintain upgrade'.\n\n" \
+          "We recommend using the update command to update to a minor version and/or\n" \
+          "operating system using 'foreman-maintain update'.\n" \
```

Comment:
I know this was already in the old text, but should we say `satellite-maintain` on Satellite installs?

There is a helper that will provide you with the right name at
https://github.com/theforeman/foreman_maintain/blob/20cdd9f7d27458137411603ea62dbad9c6e9db76/lib/foreman_maintain.rb#L168-L175